server/test/unit/server/db/test_connection.py

```diff
@@ -72,6 +72,28 @@ def test_seeds_from_config(self, mock_connect_seeds, mock_mongoengine):
                                                                             'secondhost:5678'],
                                               database)
 
+    @patch('pulp.server.db.connection.mongoengine')
+    @patch('pulp.server.db.connection._connect_to_one_of_seeds')
+    def test_multiple_seeds_no_replica_set(self, mock_connect_seeds, mock_mongoengine):
+        mock_mongoengine.connect.return_value.server_info.return_value = {'version': '2.6.0'}
+        mock_connect_seeds.return_value.server_info.return_value = {'version': '2.6.0'}
+        seeds = "firsthost:1234,secondhost:5678"
+        config.config.set('database', 'write_concern', 'majority')
+        config.config.set('database', 'seeds', seeds)
+        self.assertRaises(PulpCodedException, connection.initialize)
```

Comment:
Why is this raising an Exception? It seems that 2.6.0 with majority and a list of seeds should be allowed.